Resources/config/services.xml

```diff
@@ -50,6 +50,9 @@
         <service id="jms_translation.loader.xliff_loader" class="%jms_translation.loader.xliff_loader.class%" public="false">
             <tag name="jms_translation.loader" format="xliff" />
         </service>
+        <service id="jms_translation.loader.xliff_loader" class="%jms_translation.loader.xliff_loader.class%" public="false">
+            <tag name="jms_translation.loader" format="xlf" />
+        </service>
         
```

Comment:
Yes, But this is removed here: https://github.com/schmittjoh/JMSTranslationBundle/pull/356/commits/6c4cff5aff54cc3e0aae40de29f9cc56abc80a24